CAIPs/caip-27.md

```diff
@@ -44,8 +45,9 @@ The application would interface with a provider to make request as follows:
   "jsonrpc": "2.0",
   "method": "caip_request",
   "params": {
-    "chainId": "eip155:1",
     "session": "0xdeadbeef",
+    "scope": "eip155",
+    "chainId": "eip155:1",
```

Comment:
I thought we were removing this? It's redundant and doesn't fit the non-blockchain use cases that we spoke about. I think `session` followed with `scope` is sufficient.

Reply:
that's why I made it optional--- but then I thought about the corner case where "eip155" is the key of the scope object with multiple entries in the `chains` array, or for that matter any other namespace where (unlike polkadot or cosmos) a chainId is needed to route each RPC call...

Inelegant but fits CAIP-25's scopeObjects snugly.

Reply:
See my other comment.